scripts/circle-ci/pr_changelog_status_check.sh

```diff
@@ -12,11 +12,17 @@ fi
 cd $(dirname $0)
 
 CURRENT_REF=$(git log -1 --format="%H")
-CHANGELOG_MODIFIED=$(git diff origin/develop...HEAD --name-only | grep $CHANGELOG -q >/dev/null)$?
+CHANGELOG_COMMIT_FLAG=$(git log origin/develop..HEAD --pretty=format:%B | grep '\[no release notes\]' -q >/dev/null)$?
 
-if [ $CHANGELOG_MODIFIED -ne 0 ];
+if [ $CHANGELOG_COMMIT_FLAG -eq 0 ];
 then
-    curl -X POST -f --silent --header "Authorization: token $GITHUB_AUTH_TOKEN" -d '{"state": "failure", "description": "No modifications to release_notes.rst were found.", "context" : "changelog"}' "https://api.github.com/repos/palantir/atlasdb/statuses/$CURRENT_REF" >/dev/null
+    curl -X POST -f --silent --header "Authorization: token $GITHUB_AUTH_TOKEN" -d '{"state": "success", "description": "Bypassed with commit flag", "context": "changelog"}' "https://api.github.com/repos/palantir/atlasdb/statuses/$CURRENT_REF" >/dev/null
 else
-    curl -X POST -f --silent --header "Authorization: token $GITHUB_AUTH_TOKEN" -d '{"state": "success", "description": "release_notes.rst updated.", "context": "changelog"}' "https://api.github.com/repos/palantir/atlasdb/statuses/$CURRENT_REF" >/dev/null
+    CHANGELOG_MODIFIED=$(git log origin/develop..HEAD --name-only --pretty=format: | grep $CHANGELOG -q >/dev/null)$?
```

Comment:
Can we do the `CHANGELOG_MODIFIED` check at the same place as `CHANGELOG_COMMIT_FLAG`? then we have
```bash
if X; then
  ..
elif Y; then
  ..
else
  FAIL
fi
```